src/components/Button.js

```diff
@@ -1,8 +1,13 @@
 import './../styles/Button.css';
+import Button from '@mui/material/Button';
 
-function Button(props) {
+export default function BasicButtons(props) {
```

Comment:
Incorrect pluralisation

Reply:
Also rename the file _and_ the CSS file if you're renaming this component to preserve the mapping.

Reply:
Revert export to bottom line -Agreed upon.